src/jsMain/kotlin/dev.fritz2.kitchensink/demos/buttonDemo.kt

```diff
@@ -362,5 +363,103 @@ fun RenderContext.buttonDemo(): Div {
                 """
             )
         }
+
+        showcaseSection("Link")
+        paragraph {
+            +"There is also a special button for links which uses an "
```

Comment:
Minor wording suggestion: "There also is a special button [...]" over "There is also a special button [...]"

Reply:
Looks fine otherwise 👍🏻